Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp

```diff
@@ -48,16 +48,30 @@ namespace O3DE::ProjectManager
 
         m_repoPath = new FormLineEditWidget(tr("Remote URL"), "", this);
         m_repoPath->setMinimumSize(QSize(600, 0));
+        m_repoPath->lineEdit()->setPlaceholderText("http://github.com/example.git");
```

Comment:
```suggestion
        m_repoPath->lineEdit()->setPlaceholderText("http://github.com/o3de/example.git");
```
repository urls generally have an owner followed by the repo.git